### PR TITLE
Git submodule

### PR DESCRIPTION
**Pull request by shivani170**
Fixes: https://github.com/devtron-labs/devtron/issues/568

### CODE REVIEW

File: src/css/forms.scss [hunk 1]
```diff
@@ -65,7 +65,7 @@
 }
 
 .form__label {
-    font-size: 12px;
+    font-size: 13px;
```
Comment (manish-agrawal-ai) [inline src/css/forms.scss:68]: this impacts all the pages I assume. Is this intentional?

File: src/components/material/CreateMaterial.tsx [hunk 2]
```diff
@@ -134,14 +158,16 @@ export class CreateMaterial extends Component<CreateMaterialProps, CreateMateria
...
 
     cancel(event): void {
         this.setState({
             material: {
                 gitProvider: undefined,
                 url: '',
                 checkoutPath: '',
-                active: true
+                active: true,
+                fetchSubmodules: undefined
```
Comment (manish-agrawal-ai) [inline src/components/material/CreateMaterial.tsx:170]: this should be false (not undefined)

Comment (shivani170) [reply src/components/material/CreateMaterial.tsx:170]: fixed

File: src/css/forms.scss (hunk 1 above)
Comment (shivani170) [reply src/css/forms.scss:68]: yes

**Review by manish-agrawal-ai — APPROVED** [review]

**Review by pghildiyal — APPROVED** [review]

**Pull request merged**